packages/components/src/styles/components/tabs.scss

```diff
@@ -38,9 +68,9 @@
   position: relative;
   display: flex;
   align-items: center;
-  height: var(--token-tabs-tab-height);
+  // height: var(--token-tabs-tab-height);
   margin: 0;
-  padding: var(--token-tabs-tab-padding-vertical) var(--token-tabs-tab-padding-horizontal);
+  // padding: var(--token-tabs-tab-padding-vertical) var(--token-tabs-tab-padding-horizontal);
```

Comment:
I commented these out and moved to the new "medium" size variant styles above. I can delete them if the style structure looks ok overall or I can move them back here if that seems better to people.

Reply:
Makes totally sense to have them declared locally at "size" level in CSS